controllers/contributorController.js

```diff
@@ -122,7 +134,7 @@ const controller = {
         .filter((item) => validSkills.includes(item));
```

Comment:
As Kiong advice last night, the skills from backend should get the array of item from frontend, so should we follow that approach , so mean that in backend, we no need to handle to change from string into array